### PR TITLE
fix: tooltip trigger logic for mobile devices

### PR DESCRIPTION
**Pull request by impelcrypto**
**Pull Request Summary**

* fix: changed tooltip trigger logic for mobile devices

**Check list**
- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [x] tested on mobile devices

**This pull request makes the following changes:**

**Fixes**
* fix: changed tooltip trigger logic for mobile devices
  * ref:  https://www.notion.so/astarnetwork/Mobile-tooltip-to-be-usable-ccc245bc1bab4adeb1f7d24ba89f88e8
  * this PR won't change the UX for PC users

=Before=
The tooltip works only when users are tapping the icon on mobile devices.
https://gyazo.com/d299af7deec3c6de12a9f896b0621d2d

=After=
An opened tooltip will be closed after users tap the outside of the icon.

https://gyazo.com/b83f2e976b7759343cd43a4c77dbabc6

https://gyazo.com/abeeccf399aeee932a5deec0d95ecec8

### CODE REVIEW

**Comment by Kahonnohak** [comment]
Thank you! I can now see it in Dev tool!

- I think we can make the tooltip 12px. People can still read. 
- Texts align to the left, please.

Can we make the width narrower please?  We probably need the wider one for the next tooltip (longer) 

Can we make a rule:
- the width is 184px until texts are up to four lines
 -A after 4lines the box gets wider

https://www.figma.com/proto/NkD5tyIAQmbl6donW4eZma/Portal-V1-Kaizen?page-id=0%3A1&node-id=16%3A943&viewport=-1934%2C430%2C0.25&scaling=min-zoom

**Comment by impelcrypto** [comment]
@Kahonnohak  Updated as per your feedback. Please take a look again.

**Review by Kahonnohak — APPROVED** [review]
Thank you so much! LGTM!

File: src/components/assets/modals/ModalXcmBridge.vue [hunk 1]
```diff
@@ -172,8 +183,24 @@ export default defineComponent({
   },
   setup(props) {
     const isClosingModal = ref<boolean>(false);
+    const isMobileDisplayTooltip = ref<boolean>(false);
     const token = computed(() => props.token);
 
+    const isDisplayTooltip = computed<boolean | null>(() => {
```
Comment (bobo-k2) [inline src/components/assets/modals/ModalXcmBridge.vue:189]: I noticed this code block is repeating 3 times in different files. Can you refactor so the logic is defined only once?

**Review by bobo-k2 — CHANGES_REQUESTED** [review]

File: src/components/assets/modals/ModalXcmBridge.vue (hunk 1 above)
Comment (impelcrypto) [reply src/components/assets/modals/ModalXcmBridge.vue:189]: Sure, I will create a custom hook then.

Comment (impelcrypto) [reply src/components/assets/modals/ModalXcmBridge.vue]: @bobo-k2 
Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/451/commits/29d0b5b531e241f02561932129c2a799ade187d2) commit.

**Review by hoonsubin — APPROVED** [review]

File: src/components/assets/modals/ModalXcmBridge.vue (hunk 1 above)
Comment (impelcrypto) [reply src/components/assets/modals/ModalXcmBridge.vue]: @bobo-k2 Please help take a look my comments.

**Review by bobo-k2 — APPROVED** [review]
LGTM

**Pull request merged**